test/test_validation.py

```diff
@@ -463,6 +463,30 @@ def test_invalid_items(self):
             'Incorrectly accepts integers outside of the range'
         )
 
+   def test_valid_character_items(self):
+        self.validate_and_compare(
+            [
+                7,
+                "8",
+                8
+            ],
+            True,
+            "Does not accept integer in a string"
```

Comment:
This would be clearer as "does not accept integers provided as a string"

Reply:
Yes, better.